cmd/healthcheck/main_test.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+)
+
+func TestHealthStatusHealthy(t *testing.T) {
+	server := setUpMockCollector(t, "127.0.0.1:13133", http.StatusOK)
+	os.Args = []string{"-port=13133"}
```

Comment:
Is this necessary anymore?

Reply:
Yes not needed anymore. Updated. Thanks.